deps/chakrashim/src/v8object.cc

```diff
@@ -66,16 +66,11 @@ bool Object::Set(Handle<Value> key, Handle<Value> value) {
 
 Maybe<bool> Object::Set(Handle<Value> key, Handle<Value> value,
                         PropertyAttribute attribs, bool force) {
-  JsPropertyIdRef idRef;
```

Comment:
Since this `idRef` is used twice, is the overhead of invoking `GetOrAddPropertyRecord` twice (once per `JsObject*Property` method) smaller than the additional JSRT calls involved in fetching the id ref up front?

Reply:
no, not if it's literal or prop string. considering createString will create literalstring and presumably user will call set via a string variable that was also created on native land...